src/undeploy-actions.js

```diff
@@ -43,6 +43,32 @@ async function undeployActions (config, logFunc) {
   }
 }
 
+/** @private */
+async function getDeployedPackage (ow, packageName) {
+  if (packageName === utils.DEFAULT_PACKAGE_RESERVED_NAME) {
+    const pkg = { actions: [] }
+    const actionList = await ow.actions.list()
+    actionList.forEach(action => { // only get the actions from the default package
+      const { namespace } = utils.parsePackageName(action.name)
```

Comment:
Should we be passing `action.namespace` here and de-structuring for `name` to see if it's `default`? Was inspecting one of my test namespaces and noticed that `action.name` seems to be only the name of the action, so this call may always return `_`. 

Here's an example response from `ow.actions.list()`: 

```
  {
    annotations: [ [Object], [Object], [Object] ],
    exec: { binary: false },
    limits: { concurrency: 200, logs: 10, memory: 256, timeout: 60000 },
    name: 'generic',
    namespace: '23294-mgtestutils-stage/aio-test',
    publish: false,
    updated: 1664315558318,
    version: '0.0.4'
  }
```

Reply:
you're right, fixing

Reply:
fixed

Reply:
Should we be de-structuring for the package name, i.e `name` instead of `namespace`? I guess I'm confused if these changes are related to the default package, which I think is called `default` or the default namespace of `_`

Reply:
No - as you have observed and through my observations: `name` will always be the action name, without the package name.

So we have to rely on the `namespace` property to derive whether the action is part of a package.
If it is part of a package, `namespace` will look like `my-namespace/my-package`. 
If it is not part of the package, in the default package, `namespace` will look like this `my-namespace`.

I wish Openwhisk was more clearer in naming, but I guess we will have to work with this quirk.

Thus if `namespace` is `my-namespace/my-package`, the call to `parsePackageName` will resolve both `name` and `namespace` properties (I guess I could rename it here after dereference for clarity).
If `namespace` is `my-namespace`, the call to `parsePackageName` will resolve `name` to `_` and `namespace` to the namespace.